wpeverest/user-registration
# UR-266 Feature - Elementor widget

### All Submissions:






### Changes proposed in this Pull Request:



Previously users have to search for elementor shortcode widgets in order to display forms. This PR gives the User registration form elementor widget to display form directly.

### How to test the changes in this Pull Request:

1. Download and Install elementor plugin from wordpress.org.
2. Goto pages and click edit with elementor.
3. Goto Basic Section and search User registration form widget.
4. Drag the User registration form widget and select the form to verify it is working or not.

### Types of changes:




### Other information:




### Changelog entry

> Feature - Elementor widget.
